src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [people, setPeople] = useState<Person[] | null>(null);
+
+  useEffect(() => {
+    getPeople()
+      .then(res => {
+        setPeople(res);
+      })
+      .catch(() => {
+        //
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading ? (<Loader />) : (
+            <>
+              {!people ? (
+                <p data-cy="peopleLoadingError" className="has-text-danger">
+                  Something went wrong
+                </p>
+              ) : (
+                <>
+                  {people.length === 0 ? (
+                    <p data-cy="noPeopleMessage">
+                      There are no people on the server
+                    </p>
+                  ) : (
+                    <>
+                      <PeopleTable people={people} />
+                    </>
+                  )}
+                </>
+              )}
+            </>
```

Comment:
1. Try to avoid nested ternary operators. It's hard to read all conditions
2. You should use the fragment only if the two elements are on the same level.
3. It's better to show a block with an error if you really caught one.

Reply:
1. True. Should refactor that mess somehow.
2. My lint went crazy when I tried to leave that part without Fragment. It was sure I was missing some properties of actual jsx and I had to put some extra code there.
3. I think it depends on how the logic works. I intentionally gave people type Person | null to be able to control errors.